portfolio/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Note: try not to add new things to a commit that fixes according to comments, unless they're related.